client/src/components/Search.jsx

```diff
@@ -93,9 +266,19 @@ const Search = (props) => {
             </div>
           </div>
         )}
-      </div>
+        </div>
+    </FormGroup>
     </div>
   );
 };
 
+const buttonStyle = {
```

Comment:
One request with these -- can we turn these into classes and then put the CSS into a Search.css file or if its a global style, the application.css file?